l3embedding/train.py

```diff
@@ -81,9 +101,20 @@ def sample_one_second(audio_data, sampling_frequency, start, label, augment=Fals
             start = 0
 
     audio_data = audio_data[start:start+sampling_frequency]
+    if audio_data.shape[0] != sampling_frequency:
+        # Pad audio that isn't one second
+        warnings.warn('Got audio that is less than one second', UserWarning)
```

Comment:
what's the difference between using `warnings` and `logging.warning`?

Reply:
`warnings` deals with Python warnings, whereas `logging.warning` logs something to the WARNING logging level